solr/core/src/java/org/apache/solr/util/stats/MetricUtils.java

```diff
@@ -827,7 +845,7 @@ public static <T extends PlatformManagedObject> void addMXBeanMetrics(
       try {
         final Class<? extends PlatformManagedObject> intf =
             Class.forName(clazz).asSubclass(PlatformManagedObject.class);
-        MetricUtils.addMXBeanMetrics(obj, intf, null, consumer);
+        MetricUtils.addMXBeanMetrics(obj, intf, prefix, consumer);
```

Comment:
What's this about here; seems like metrics would be published in a different spot, right?

Reply:
I forgot to comment on this when creating the pull request.

I think this was a typo when this code was initially added. The `prefix` parameter of the method is not passed through to the other method. In practice, all callers to `addMXBeanMetrics()` in Solr code set null for this parameter, so this will not change anything.

Another option could be to just remove the parameter, but I think it's better to keep it if we have later more categories for metrics.